chart/values.yaml

```diff
@@ -2,6 +2,10 @@
 # which run inside the prometheus-benchmark - e.g. vmaget, vmalert, vmsingle.
 vmtag: "v1.77.1"
 
+# nodeSelector is the list of key-value pairs for selecting instances
+# where benchmark pods should be deployed.
+nodeSelector: {}
```

Comment:
The nodeSelector mist be optional. If it isn't specified, then k8s should be able to run benchmark-related pods at any nodes.

Reply:
made it optional